Codeinwp/neve
# fix: search component inside sidebar [closes #2170]

### Summary
Forces search component to floating when inside the sidebar row of the header.


### Will affect visual aspect of the product

NO

### Screenshots 
![image](https://user-images.githubusercontent.com/15010186/97418758-dabeae00-1911-11eb-8946-658e9a754311.png)

### Test instructions


- Add the search component inside the sidebar
- It should be forced to floating regardless of the selected Open Behaviour
- It should work fine


Closes #2170 .
